src/tests/state/experiments/selectors.test.js

```diff
@@ -120,6 +122,28 @@ describe("getFilteredExperimentsByDate tests", () => {
         endDate: "2018-07-22"
       }
     });
-    expect(getFilteredExperimentsByDate(mockedState)).toEqual(fromJS([]));
+    expect(getFilteredExperiments(mockedState)).toEqual(fromJS([]));
+  });
+
+  it("should return experiment that has type `pref`", () => {
+    const mockedState = fromJS({
+      experiments: mockedExperiments, 
+      type: {
+        selectedType: "pref"
+      }
+    });
+
+    expect(getFilteredExperiments(mockedState)).toEqual(fromJS([experiment1]));
+  });
+
+  it("should return experiment that has type `addon`", () => {
+    const mockedState = fromJS({
+      experiments: mockedExperiments, 
+      type: {
+        selectedType: "addon"
+      }
+    });
+
```

Comment:
What about a test for all types, ie pref and addon?

Reply:
Good point, added that test! My default state for selected type is currently null, but in the component it's set to "" if we're filtering for all types. Should I change default state to "" for consistency?

Reply:
Oh I see yeah, you can't put `null` in a DOM element, I like `null` better for unset values, but then you have the case that sometimes it'll be `null` and sometimes it'll be `""` depending on whether the app just initialized or if you set it back to 'all'.  Yeah switch it to `""` for consistency 👍

Reply:
> you can't put null in a DOM element

why not?

Reply:
^ oh whoops late reply sorry I missed this in the sea of email notifications!

Is something that you posted in my other PR the way to set tags null? By not setting a value property?